docs/install/install-eventing-with-yaml.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "Installing Knative Eventing using YAML files"
+linkTitle: "Install Eventing using YAML"
+weight: 03
+type: "docs"
+showlandingtoc: "false"
+---
+
+This topic describes how to install Knative Eventing.
+You can install the Knative Eventing component by applying YAML files using the `kubectl` CLI.
+
+When installing Knative, you can install the Serving component, Eventing component, or both on your cluster.
+For how to install the Serving component, see [Installing Serving using YAML files](./install-serving-with-yaml.md)
+
+## Prerequisites
+
+Before installation, you must meet the prerequisites.
+See [Knative Prerequisites](./prerequisites.md).
+
+
+## Install the Eventing component
```

Comment:
Installing? Can we standardize on using gerunds in headings maybe?

Reply:
I'm used to using imperatives in headings. I can't find a rule in the Google style guide and the Knative docs are not consistent on this. I usually revert to what I'm used to if I cannot find a rule. 

Is there a forum where we can discuss style decisions such as these so that we can all be doing the same thing?